docs/configuration/overview.mdx

```diff
@@ -60,6 +60,31 @@ melos:
   sdkPath: .fvm/flutter_sdk
 ```
 
+## useRootAsPackage
+
+Whether to include the repository root as a package in the workspace.
+
+When enabled, the root directory (containing the workspace configuration) will be treated as a package and included in workspace operations such as scripts, filtering, and categorization.
```

Comment:
We don't have any markdown linter in effect currently, but I think that we keep to max 80 line length.